--- app/mainScreenPreload.js	2021-07-02 10:38:54.516373139 +0200
+++ app/mainScreenPreloadPatched.js	2021-07-02 10:39:04.333263069 +0200
@@ -58,6 +58,9 @@
     (_uncaughtExceptionHan = uncaughtExceptionHandler) === null || _uncaughtExceptionHan === void 0 ? void 0 : _uncaughtExceptionHan(err, origin);
   });
   window.popouts = new Map();
+
+  const UnreadBellLib = require("$UNREAD_PATCH_DIR/lib.js");
+  contextBridge.exposeInMainWorld("UnreadBellLib", UnreadBellLib);
 } else {
   window.addEventListener('load', _ => {
     window.opener.popouts.set(window.name, window);
